CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 ## Changelog
 
+### 2.86
+
+Release date: 2020-11-18
+
+* Fix: Replace use of NamedArgsAndClosure as return value for invokeDescribable. ([JENKINS-64185](https://issues.jenkins-ci.org/browse/JENKINS-64185))
```

Comment:
nit: I think it would be helpful to explicitly mention that this is fixing a regression in version 2.85 related to Declarative Pipelines.

```suggestion
* Fix: Revert changes to usage of `NamedArgsAndClosure` class in version 2.85 that caused various regressions for Declarative Pipelines. ([JENKINS-64185](https://issues.jenkins-ci.org/browse/JENKINS-64185))
```